cmd/controller/app/controller.go

```diff
@@ -239,7 +238,8 @@ func buildControllerContext(ctx context.Context, stopCh <-chan struct{}, opts *o
 			DefaultAutoCertificateAnnotations: opts.DefaultAutoCertificateAnnotations,
 		},
 		CertificateOptions: controller.CertificateOptions{
-			EnableOwnerRef: opts.EnableCertificateOwnerRef,
+			EnableOwnerRef:            opts.EnableCertificateOwnerRef,
+			RenewBeforeExpiryDuration: opts.RenewBeforeExpiryDuration,
```

Comment:
Moved to CertificateOptions which seemed more appropriate as it's now used by one of the certificate controllers.